numba/cuda/cudadrv/driver.py

```diff
@@ -1735,10 +1751,18 @@ def __int__(self):
         return self.handle.value or 0
 
     def __repr__(self):
-        if self.handle.value:
-            return "<CUDA stream %d on %s>" % (self.handle.value, self.context)
+        default_streams = {
+            0: "<Default CUDA stream on %s>",
+            1: "<Legacy default CUDA stream on %s>",
+            2: "<Per-thread default CUDA stream on %s>",
+        }
```

Comment:
Should this use this:  https://github.com/numba/numba/pull/5347/files#diff-511a1676aeb16a8663dbcddcd2d59949R22-R24

Reply:
It should, and it would also be good sense for me to add `CU_STREAM_LEGACY` for consistency too.

Reply:
Added now.